### DISCUSSION TITLE
I used Pyodide for a web interface for my scientific code

### DISCUSSION THREAD

**Discussion by Kodiologist** [pyodide/pyodide#2291] [Show and tell]
[Artiruno](https://github.com/Kodiologist/Artiruno) is a Python program for verbal decision analysis that used to have a web interface in [Brython](https://brython.info/). Changing to Pyodide noticeably increased speed, and gives me some peace of mind that substantially the same code is running for the web interface as for the command-line interface. It also enabled running the test suite in a web browser with pytest. Formerly, I'd had to kludge together my own pytest replacement.

**Comment by ryanking13** [comment]
Thanks for sharing your Pyodide usage with us! We gladly accept any kind of feedbacks on using Pyodide :)
